### DISCUSSION TITLE
CPU load

### DISCUSSION THREAD

**Discussion by Martin363363** [outdoorbits/little-backup-box#247] [General]
Hi!
Thanks for the great work with LBB!
I'm just wondering if my CPU loads seems reasonable. I get around 45% when LBB is just idling (raspberry pi 4b, 4GB). Have tried both stable and development version with similar results.

**Comment by outdoorbits** [comment]
Did you configure a standard backup? Is LBB waiting for a device?

**Reply by Martin363363** [reply]
Ahh, thats it. I do have a standard backup (USB-USB). Turned that of and CPU load decreased to 5-10%. However not having the standard backup makes the system less user friendly. I'm having trouble getting the backup started so I think I'll get better cooling and accept high CPU load.

**Comment by outdoorbits** [comment]
Do you have a relevant delay between booting and connecting the device? Lbb checks frequently for new devices, that's the high CPU load. When backup starts, this part is finished and doesn't take CPU load anymore.

**Reply by Martin363363** [reply]
So, I've tried a few different alternatives and the automatic detection of drives works good, and as you say the CPU load decreases while copying (to around 30%). 

However I've now run into another problem which I can´t seem to solve. The backup stops (transfers a couple of files) with the following error "Exception: Error code '23', Partial transfer due to error". I've reinstalled LBB, reformated both source card and backup drive (exFAT on both) with the same result. Any ideas?

**Reply by outdoorbits** [reply]
Today I changed the reporting system and connected to this I fixed an issue about retrying. If you won't find any errors on your SSD, maybe now one of the next tries does the job.

**Comment by outdoorbits** [comment]
I've seen this error code some time ago also. It is a partial transfer 
error. In my case, when I checked the devices by Tools - Repair - check 
I always found an error and I could fix it by the repair function. Are 
you using a powered USB hub in this tests?
